source/20_spec/config_spec.rst

```diff
@@ -0,0 +1,154 @@
+      hopper_ppo_config.py should be changed to hopper_onppo_config.py。
+
+-  exp_name field
+
+   -  main_config must include exp_name filed
+
+   -  The naming convention is environemnt+algorithm+seed,such as \ ``qbert_sqil_seed0``
+
+-  Name of the file path
+
+   -  See the sqil example, commented accordingly.If multiple models need to be loaded, the model path (model_path) variable is named as follows：prefix1_model_path,prefix2_model_path,...,
+      varibles of data_path are named in same way.
+
+.. code:: python
+
+   config = dict(
+       ...,
+       # Users should add their own model path here. Model path should lead to a model.
+       # Absolute path is recommended.
+       # In DI-engine, it is ``exp_name/ckpt/ckpt_best.pth.tar``.
+       model_path='model_path_placeholder',
+       # Users should add their own data path here. Data path should lead to a file to store data or load the stored data.
+       # Absolute path is recommended.
+       # In DI-engine, it is usually located in ``exp_name`` directory
+       data_path='data_path_placeholder',
+   )
+
+Main Specification
+~~~~~~~~~~~~~~~~~~~~
+
+-  For env_manager field in create_config, except for simple environments 
+   cartpole, pendulum, bitflip 
+   environment uses base, other environments normally use subprocess：
+
+   .. code:: python
+
+      env_manager=dict(type='subprocess'),
+
+-  Ensure evaluator_env_num：n_evaluator_episode = 1:1 （expect smac environment）
+
+-  manager field shoudl generally not be included in the env field of main_config
+   (shared_memory defaults to True when the manager field is not included)：
+
+   -  smac environment is an exception,due to the state dimension problem,smac needs to set shared_memory=Fasle。
+
+   -  In environments other than the smac environment, if an error is reported due to the state dimension problem,you can include manager field and set  shared
+      memory=False。
+
+-  If you want to turn on/off shared memory, please control it in env.manager filed
+
+   .. code:: python
+
+      config = dict(
+          ...,
+          env=dict(
+              manager=dict(
+                  shared_memory=True,
+              ),
+          ),
+      )
+
+-  create config
+
+   -  in env field,only ``type`` 和 ``import_names``\ two fields,
+      Such as：
+
+   .. code:: python
+
+      env=dict(
+          type='atari',
+          import_names=['dizoo.atari.envs.atari_env'],
+      ),
+
+   -  nnormally field \ ``replay_buffer``\ is unnecessary。If you want to use the buffer stored as deque，please specify the type of replay_buffer：
+
+      .. code::
+
+         replay_buffer=dict(type='deque'),
+
+-  serial_pipeline
+
+   -  Please apple secondary references to avoid circular
```

Comment:
Please utilize two-level import to avoid circular import：specifically, use `from ding.entry import serial_pipeline`  instead of  `from ding.entry.serial_entry import serial_pipeline`.